guides/gatherers.md

```diff
@@ -66,3 +67,39 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+### sbd_config
+
+**Name:** `sbd_config`
+
+This gatherer allows accessing information contained in `/etc/sysconfig/sbd`
+
+```
+sbd_related_fact = sbd_config(some.argument)
```

Comment:
I would remove this code snippet as well. I don't find major value, and it is duplicated in all the gatherers.
There is already the generic example in the top